src/NEPSolver.jl

```diff
@@ -118,18 +118,21 @@ Executes z if displaylevel>1.
         A=v->compute_Mlincomb(nept,complex(conj(λ)),compute_Mlincomb(nep,complex(λ),complex(v)))
 
         # initialization
-        x/=norm(x); v=A(x); ρ=x⋅v; q=zeros(ComplexF64,size(nep,1));
-        k=1; err=1; tol=1e-12
+        normalize!(x); v=A(x); ρ=x⋅v; q=zeros(ComplexF64,size(nep,1));
+        k=1; err=one(eltype(x)); tol=1e-12
         while (k<maxit)&&(err>tol)
-          g=v-ρ*x;
-          aa=[x -g q]'*[v -A(g) A(q)]; aa=(aa+aa')/2;
-          mm=[x -g q]'*[x -g q]; mm=(mm+mm')/2;
+          g = v-ρ*x;
+          xgq = [x -g q]

```

Comment:
Extract this instead of creating it multiple times.

Reply:
:+1: